core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/rules/TSqlCallMapper.scala

```diff
@@ -8,10 +8,42 @@ class TSqlCallMapper extends CallMapper {
     call match {
       case CallFunction("DATEADD", args) =>
         processDateAdd(args)
+      case CallFunction("GET_BIT", args) => BitwiseGet(args.head, args(1))
+      case CallFunction("SET_BIT", args) => genBitSet(args)
+      case CallFunction("CHECKSUM_AGG", args) => checksumAgg(args)
       case x: CallFunction => super.convert(x)
     }
   }
 
+  /**
+   * Converts a CHECKSUM_ARG call to a MD5 function call sequence
+   * @param args
+   * @return
+   */
+  private def checksumAgg(args: Seq[Expression]): Expression = {
+    CallFunction("MD5", Seq(CallFunction("CONCAT_WS", Seq(Literal(","), CallFunction("COLLECT_LIST", args)))))
```

Comment:
```suggestion
    Md5(ConcatWs(Literal(","), CollectList(args)))
```
Can we use concrete IR nodes when remapping, instead of generic CallFunction? same for genBitSet. otherwise it's inconsistent.

Reply:
Let me look again. I tried that first  but because of some call sequence it ended up not translating and now I do not remember why. So I switched back to CallFunction, which then translates as it goes back through the mapper. I agree with you though - let me find out why.

Reply:
Mappers fixed to pass through already resolved conrete functions and CallFunction IR replaced with concrete classes.